pkg/network/usm/usm_http_monitor_test.go

```diff
@@ -281,15 +286,10 @@ func TestGoTLSMapCleanup(t *testing.T) {
 		}
 	}
 
-	require.Eventually(t, func() bool {
-		goTLSMapFinal := utils.CountMapEntries(t, goTLSMap)
-		goTLSConnByTupleMapFinal := utils.CountMapEntries(t, goTLSConnByTupleMap)
-		if goTLSMapFinal != 0 {
-			t.Logf("Map goTLSMap still has %d entries", goTLSMapFinal)
-		}
-		if goTLSConnByTupleMapFinal != 0 {
-			t.Logf("Map goTLSConnByTupleMap still has %d entries", goTLSConnByTupleMapFinal)
+	require.EventuallyWithT(t, func(collect *assert.CollectT) {
+		for _, m := range mapsInstances {
+			count := utils.CountMapEntries(t, m)
+			assert.Zero(collect, count, "map %s should be empty after proxy exit", m.String())
 		}
-		return goTLSMapFinal*goTLSConnByTupleMapFinal == 0
 	}, 5*time.Second, 100*time.Millisecond, "map should be empty after proxy exit")
```

Comment:
```suggestion
	}, 5*time.Second, 100*time.Millisecond, "maps should be empty after proxy exit")
```

Reply:
It depends on the failure, may be also single map :)